frontend/packages/client/.prettierrc

```diff
@@ -7,13 +7,14 @@
   "semi": true,
   "bracketSpacing": true,
   "importOrder": [
-    "react",
+    "^react*",
     "contexts",
     "components",
     "hooks",
     "const",
     "utils",
     "<THIRD_PARTY_MODULES>",
+    "^@*$",
```

Comment:
added this to handle imports with '@....' I'll create a separate PR to do:  
```
    "^react*",
    "<THIRD_PARTY_MODULES>", ==> external packages after all react related imports 
    "^@*$", => imports with  @
    "^contexts*",
    "^components*",
    "^hooks*",
    "const",
    "utils",
```
this will create a lot of changes in the import order